src/legacyapi/apirequests.ts

```diff
@@ -32,35 +32,39 @@ export async function getJson<Request extends BaseRequest>(path: string, request
     req = {
       headers: {
         "Accept": "application/json",
-        "x-accesstoken": accessToken,   // x-accesstoken Kleinschreibung erforderlich
+        "x-accesstoken": accessToken,   // X-accesstoken Kleinschreibung erforderlich
         "authorization": accessToken
       } as any
     };
   }
   try {
-    let response = await fetch(url, req);
+    const response = await fetch(url, req);
     switch (response.status) {
-      case 200:
-        let json = await response.json();
+      case 200: {
+        const json = await response.json();
         if (json.result !== ApiResult.API_OK) {
           console.log("getJson " + url + ": " + json.result);
           console.log(json);
         }
         return json;
-      case 403:  // API_FORBIDDEN -> server requests redirect to signin
-        if (typeof window !== "undefined"  //  not possible on server rendering
+      }
+      case 403: {  // API_FORBIDDEN -> server requests redirect to signin
+        if (typeof window !== "undefined"  //  Not possible on server rendering
           && !window.location.pathname.startsWith("/signin")) {
           window.location.href = "/signin?redirect=" + encodeURIComponent(window.location.pathname);
         }
-      default:
+        break;
```

Comment:
I am not sure about this - was this break intentionally missing as fall-through?